cdap-ui/app/cdap/components/EntityCard/EntityCardHeader/index.js

```diff
@@ -55,7 +54,10 @@ export default class EntityCardHeader extends Component {
                 className={classnames("entity-card-header", this.props.className)}
               >
                 <h4>
-                  <span className={classNames('entity-icon', this.props.entity.icon)}></span>
+                  <SVG
```

Comment:
`<FontContainer>` is a little bit long. Do you think it would be a good idea to name it something like `<Icon value="icon-fist">`? As `icon` is not an HTML element.